arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -446,6 +446,9 @@ public void resolveAtContextReEntry(Hub hub, CallFrame frame) {
         checkState(successBit);
         scenarioFragment.setScenario(CALL_EOA_SUCCESS_WONT_REVERT);
         emptyCodeFirstCoupleOfAccountFragments(hub);
+        long gasAfterCall = frame.frame().getRemainingGas();
+        commonValues.gasNext(gasAfterCall);
+        hub.currentFrame().lastValidGasNext(gasAfterCall);
```

Comment:
For EOA's and SMC's we grab the `gasRemaining` in the frame when it resumes execution. This fixes the gas.